src/mono/msvc/libmonoruntime.vcxproj

```diff
@@ -97,8 +97,8 @@
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <PreprocessorDefinitions>WIN32;WIN32_LEAN_AND_MEAN;_LIB;$(GC_DEFINES);_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <AdditionalIncludeDirectories>$(MONO_DIR);$(MONO_INCLUDE_DIR);$(LIBGC_CPPFLAGS_INCLUDE);$(GLIB_CFLAGS_INCLUDE);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>WIN32;WIN32_LEAN_AND_MEAN;_LIB;$(GC_DEFINES);_DEBUG;%(PreprocessorDefinitions);TARGET_WINDOWS;PALEXPORT=extern "C" __declspec(dllexport)</PreprocessorDefinitions>
```

Comment:
Move this into a property that we can add to the preprocessor, like SHIM_GLOBALIZATION_DEFINES or similar.

Reply:
Also add the property just after the GC_DEFINES.

Reply:
Then you could only set the property value if we do netcore build in mono.props.

Reply:
Done